db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # convert load_balancer_port and instance_port into ranges, preserving
+        # existing values
+        execute <<-SQL
+          ALTER TABLE load_balancer_listeners
+            ALTER COLUMN load_balancer_port SET DATA TYPE int4range
```

Comment:
Can you describe some of the intent of using arel? I don't actually see many examples of how to use arel to convert data (especially with a postgres specific type constructor).  I'm not sure either that the created arel code would be easier to read than the raw SQL to be executed.